### ISSUE TITLE
Why this package on the pub.dev was marked as [UNIDENTIFIED]?

### ISSUE THREAD

**Issue by bettersun** [rrousselGit/freezed#708]
**Why this package on the  pub.dev was marked as [UNIDENTIFIED]?**
https://pub.dev/packages/freezed
https://pub.dev/packages/freezed/score

Because of the [UNIDENTIFIED] mark of this package on the  pub.dev ,
I don't know should I use this package or not.

I used this package for a short time before, and  I think this is an awesome package.

**Comment by SunlightBro** [comment]
This is an issue with [pana](https://pub.dev/packages/pana) the package analysis tool for pub dev
This repo hosts 3 packages:
- [`_internal`](https://github.com/rrousselGit/freezed/blob/master/packages/_internal/pubspec.yaml)
- [`freezed`](https://github.com/rrousselGit/freezed/blob/master/packages/freezed/pubspec.yaml)
- [`freezed_annotation`](https://github.com/rrousselGit/freezed/blob/master/packages/freezed_annotation/pubspec.yaml)

For ease of use during development `freezed` uses a `pubspec_overrides.yaml` that points to the local freezed_annotation packages (instead of the hosted one). 

`pana` can't handle that, and throws during the **✗ Pass static analysis (0 / 30)** step and the following steps

You can reproduce this by running
```sh
dart pub global activate pana
pana --hosted freezed
//this will only give freezed 40/120 points
git clone https://github.com/rrousselGit/freezed.git
pana freezed/packages/freezed
//this will give freezed 100/120 points
```

### TLDR
You can use freezed, no problem. pana just can't evaluate freezed properly at the moment.

**Comment by rrousselGit** [comment]
I unpublished the pubspec_overrides file. This shouldn't be a problem now.

**Comment by bettersun** [comment]
Thank you.
